config/application.rb

```diff
@@ -10,6 +10,7 @@ module Target
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 7.0
+    config.autoloader = :classic
```

Comment:
We shouldn't do this. Rails 6 comes by default with Zeitwerk, and we should use that Autoloader

Reply:
It was fix with commit